lib/tasks/asset_manager.rake

```diff
@@ -28,9 +29,10 @@ namespace :asset_manager do
   private
 
   def usage_string
-    %{Usage: asset_manager:migrate_assets[<path>]
+    %{Usage: asset_manager:migrate_assets[<path>,<draft>]
 
       Where <path> is a subdirectory under Whitehall.clean_uploads_root e.g. `system/uploads/organisation/logo`
+      And <draft> is an optional Boolean representing whether the uploaded asset should be in the draft state
```

Comment:
Very minor: maybe note default is `false` and that `true` should be represented by the string `'true'`. I might've expected `'1'` to work, but I don't think it will.

Reply:
Note that I've since removed the commit that introduced this change.